### ISSUE TITLE
[QUESTION] <Any plans to add compatibility with pokemon based mobs such as pixelmon or cobblemon?>

### ISSUE THREAD

**Issue by Xboxgamer222** [Darkhax-Minecraft/BotanyPots#397]
### Question

Any plans to add compatibility with pokemon based mobs such as pixelmon or cobblemon? For example, the apricorn spouts in cobblemon and the like? just wanted to ask :)

**Comment by Darkhax** [comment]
Yes, I just haven't been able to find the time to add support for newer versions. It's on my TODO list though.
